WalletWasabi.Fluent/ViewModels/Wallets/VerifyRecoveryPhraseViewModel.cs

```diff
@@ -0,0 +1,138 @@
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using DynamicData;
+using DynamicData.Binding;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Blockchain.Keys;
+using WalletWasabi.Fluent.Validation;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Authorization;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+using WalletWasabi.Logging;
+using WalletWasabi.Models;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets;
+
+[NavigationMetaData(Title = "Recovery Words Verification")]
+public partial class VerifyRecoveryPhraseViewModel : RoutableViewModel
```

Comment:
IMO even the class should be renamed to `VerifyRecoveryWordsViewModel`.

Reply:
done.